cctk/orca_file.py

```diff
@@ -17,41 +17,29 @@ class OrcaJobType(Enum):
     All jobs have type ``SP`` by default.
     """
 
-    SP = "sp"
+    SP = ("sp", ["energy", "scf_iterations"])
     """
     Single point energy calculation.
     """
-    OPT = "opt"
-     # should include looseopt, tightopt, normalopt, verytightopt, copt, zopt, GDIIS-COPT, GDIIS-ZOPT, GDIIS-OPT
-    SCANTS = "scants"
-    OPTTS = "optts"
-    TIGHTOPT = "tightopt"
+
+    OPT = ("opt", ["rms_gradient", "rms_step", "max_gradient", "max_step"])
     """
     Geometry optimization.
     """
 
-    FREQ = "freq" # should include AnFreq and NumFreq 
+    FREQ = ("freq", ["gibbs_free_energy", "enthalpy", "frequencies", "temperature"])
     """
     Hessian calculation.
     """
 
-    NMR = "nmr"
+    NMR = ("nmr", ["isotropic_shielding"])
```

Comment:
Please also parse the paramagnetic, diamagnetic, and total shielding tensors as n_atoms x 3 x 3 array.  Please read the anisotropy for each atom as well.